planex/cmd/clone.py

```diff
@@ -116,25 +116,29 @@ def main(argv=None):
     """
     Entry point
     """
-    args = parse_args_or_exit(argv)
+    _args = parse_args_or_exit(argv)
 
-    for pinpath in args.pins:
-        pin = Link(pinpath)
+    # for pinpath in args.pins:
+    #     pin = Link(pinpath)
 
-        if args.jenkins:
-            print('echo "Cloning %s"' % pin.url)
-            clone_jenkins(pin.url, args.repos, pin.commitish, args.credentials)
+    #     if args.jenkins:
```

Comment:
Are these going to be re-added at some point?

Reply:
Yes, there is an open issue to re-write `clone.py`